hw12task1.py

```diff
@@ -0,0 +1,82 @@
+"""
+Homework.
+"""
+# Колода карт.
+# Напишите программу которая содержит список карт, умеет
+# их перемешивать и позволяет пользователю достать карту
+# из колоды по ее номеру. Всего в колоде 54 карты. Класс
+# Card содержит спискок номеров карт и список мастей.
+# class Card:
+#     number_list = ...
+#     mast_list = ...
+
+#     def __init__(self):
+#         ...
+
+# class CardsDeck:
+#     def __init__(self):
+#         ...
+
+# deck = CardsDeck()
+# deck.shuffle()
+# card_number = int(input('Выберите карту из колоды в 54 карты:'))
+# card = deck.get(card_number)
+# print(f'You card is: {card}')
+# >> Hearts 10
+
+# card_number = int(input('Выберите карту из колоды в 54 карты:'))
+# card = deck.get(card_number)
+# print(f'You card is: {card}')
+# >> Diamonds 6
+
+
+import random
+
+
+class Card:
+    """
+    Contains a list of card numbers and a list of suits.
+    """
+    number_list = ['2', '3', '4', '5', '6', '7', '8', '9', '10', 'Jack', 'Queen', 'King', 'Ace']  # pylint: disable=line-too-long  # noqa: E501
```

Comment:
It should be in .pylintrc or pylint workflow file
```suggestion
    number_list = ['2', '3', '4', '5', '6', '7', '8', '9', '10', 'Jack', 'Queen', 'King', 'Ace']
```

Reply:
Removed